contrib/platform/src/com/sun/jna/platform/mac/CoreFoundation.java

```diff
@@ -420,7 +477,7 @@ CFMutableDictionaryRef CFDictionaryCreateMutable(CFAllocatorRef alloc, int capac
      *            was present).
      * @return {@code true} if a matching key was found, otherwise {@code false}.
      */
-    boolean CFDictionaryGetValueIfPresent(CFDictionaryRef theDict, CFStringRef key, PointerType value);
+    boolean CFDictionaryGetValueIfPresent(CFDictionaryRef theDict, PointerType key, PointerByReference value);
```

Comment:
I aggree with the change of the `value` parameter, I would keep the key as `CFStringRef`.

Reply:
Ignore this, as it should follow the discussion areound `CFDictionaryGetValue`